### PR TITLE
 Publish DeviceUpdateEvent message

### PR DESCRIPTION
**Pull request by jerrytrieu**
Depends on https://github.com/advancedtelematic/libats/pull/85

### CODE REVIEW

File: src/main/scala/com/advancedtelematic/director/http/AdminResource.scala [hunk 1]
```diff
@@ -248,45 +236,10 @@ class AdminResource(extractNamespace: Directive1[Namespace], val keyserverClient
       (path("images") & get) {
         listInstalledImages(ns, device)
       } ~
-      // Deprecated in favor of "assignments/:deviceId" endpoint
-      pathPrefix("queue") {
```
Comment (doanac) [inline src/main/scala/com/advancedtelematic/director/http/AdminResource.scala]: I find it a little annoying that this project uses *versioned* APIs: https://github.com/advancedtelematic/director/blob/master/src/main/scala/com/advancedtelematic/director/http/DirectorRoutes.scala but then decides to make API changes that will break people's tooling that depends on these APIs.

Comment (jerrytrieu) [reply src/main/scala/com/advancedtelematic/director/http/AdminResource.scala]: Understood, will redo this. Thanks!

File: src/main/scala/com/advancedtelematic/director/db/SetMultiTargets.scala [hunk 2]
```diff
@@ -93,7 +93,12 @@ class SetMultiTargets()(implicit messageBusPublisher: MessageBusPublisher) exten
 
     db.run(dbAct.transactionally).flatMap { scheduled =>
       Future.traverse(scheduled){ device =>
-        messageBusPublisher.publish(UpdateSpec(namespace, device, UpdateStatus.Pending))
+        messageBusPublisher.publish(DeviceUpdateEvent(
```
Comment (houcros) [inline src/main/scala/com/advancedtelematic/director/db/SetMultiTargets.scala]: Would it make sense to use `publishSafe` here? Or we can move on if we fail to publish the message?

Comment (houcros) [reply src/main/scala/com/advancedtelematic/director/db/SetMultiTargets.scala]: I just realised there's many `publish` like this. I guess they're not critical.

**Pull request closed without merging**